### DISCUSSION TITLE
Having Problems mit KeyClu when using Adobe Apps and holding CMD + OPT + SPACE to zoom in

### DISCUSSION THREAD

**Discussion by Lomaxxxxx** [Anze/KeyCluCask#59] [Q&A]
Hey everyone,

i switched from KeyCue (paid user) to KeyClu yesterday, because i like to support non commercial apps and trying to help them grow.

As a graphic designer i use different apps from adobe daily and heavy, so i am using many times a day the shortcut CMD + SPACE TO activate the zoom-in tool and CMD + OPT + SPACE to zoom out.

While holding the keys down the shortcuts panel drops in and overlay my app, while the zoom tool is still working. when i stop pushing the keys the shortcuts panel stays in the window and i have to get into the menubar to hid it or press again CMD key. 

anyone here have any solution for this besides raising the delay time? 

thank you

**Comment by Anze** [comment]
Hey [Lomaxxxxx](https://github.com/Lomaxxxxx),
Sorry to hear you're having trouble. The last version had an issue that might show overlay when it shouldn't. Can you please check the latests version?

**Reply by Lomaxxxxx** [reply]
Downloaded the latest update and it works. Awesome and lighting fast reply speed. Sending you a big coffee now ;-)
Thank you

**Reply by Lomaxxxxx** [reply]
Hey there, since the last update it is way better, but i got a request (if this is possible).
It is still the same situation, but the windows wont stay as an overlap, like before the update. Thats fine

It would be a dream if you could add the behavior, that the KeyClu App doesnt pop up when there is an additional key pressed to the CMD key

Example:

I told above that i am using very often the zoom tool in photoshop. You have to press CMD + SPACE + CTRL to activate the Zoom Out and CMD + SPACE to zoom in. This is for most of the adobe apps the same.

It would be so great if there is a possibility that KeyClu doesnt pop up, as long there is an additional key to the CMD pressed

do you think that is doable?

**Reply by Anze** [reply]
Hey [Lomaxxxxx](https://github.com/Lomaxxxxx),
The problem you're describing shouldn't occur, as it's designed to function as you expected. I'll try to reproduce and investigate why this is happening. To proceed, I need some information from you: could you please share the keys you've set for activation and the delay set for holding the key? The most convenient way would be to provide a screenshot or copy-paste the output of a terminal command: `defaults read com.0804Team.KeyClu`.

**Reply by Lomaxxxxx** [reply]
Hey,

this is from the terminal command

```
    "NSWindow Frame SUUpdateAlert" = "446 420 620 398 0 0 1512 944 ";
    SUAutomaticallyUpdate = 1;
    SUEnableAutomaticChecks = 1;
    SUHasLaunchedBefore = 1;
    SULastCheckTime = "2024-03-07 09:17:36 +0000";
    SUSendProfileInfo = 0;
    SUUpdateGroupIdentifier = 2968791361;
    activationKeyType = 2;
    hideMenuIcon = 1;
    highlightAccentColor = 1;
    keyHoldingDelay = "0.75";
    launchAtLogin = 1;
    showBookmarks = 1;
    silentLaunchQuit = 1;
```

**Reply by Anze** [reply]
Thanks for sharing the config. It's really helpful.

**Reply by Anze** [reply]
Hey [Lomaxxxxx](https://github.com/Lomaxxxxx),
I've just made a new release, where fixed this issue.
